app/services/service_creator.rb

```diff
@@ -37,12 +37,8 @@ def save!(params)
   end
 
   def save_default_backend_api(params)
-    if provider_can_use?(:api_as_product)
-      return true if %i[path private_endpoint].none? { |key| params.key?(key) }
-      backend_api_proxy.update!(params)
-    else
-      backend_api_proxy.update!(private_endpoint: BackendApi.default_api_backend)
-    end
+    return true if %i[path private_endpoint].none? { |key| params.key?(key) }
```

Comment:
If this comment is correct: https://github.com/3scale/porta/pull/2190/files#diff-437d3143cf330769c5e83c1d9385f74182d8ef63f03d4024af7ee6b399a6af0dR41
then it means that this field is not used anywhere, then:
```suggestion
    return true params.key?(:path)
```